runtime/lua/vim/lsp/log.lua

```diff
@@ -80,9 +81,10 @@ end
 for level, levelnr in pairs(log.levels) do
   -- Also export the log level on the root object.
   log[level] = levelnr
-end
 
-vim.tbl_add_reverse_lookup(log.levels)
+  -- Add a reverse lookup.
```

Comment:
My point was actually to _not_ shoehorn the reverse lookup into the _same_ table. Can we make this a separate table?

Reply:
(And same below)

Reply:
@clason please let me know if I've addressed your concerns :)

Reply:
looks good!